src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1124,9 +1124,9 @@ addFromClauseForLogicalDatabaseDump(PQExpBuffer buf, TableInfo *tbinfo)
 						  "ON a.database_name = b.name COLLATE \"C\" "
 						  "WHERE b.dbid = %d "
 						  "AND a.rolname NOT IN "
-						  "('master_dbo', 'master_db_owner', 'master_guest', "
-						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', "
-						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest') ",
+						   "('master_dbo', 'master_db_owner', 'master_guest', 'master_db_datareader', 'master_db_datawriter', "
+						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', 'msdb_db_datareader', 'msdb_db_datawriter', "
+						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest', 'tempdb_db_datareader', 'tempdb_db_datawriter') ",
```

Comment:
This has been refactored in a separate static char array in this [PR](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/pull/447/files#diff-80f19b63ae5d1d8e5293fc3f35fbd1f33b77cde701221ceed9eccca3f43727f0R55). We need to accommodate those changes.